### DISCUSSION TITLE
Unable to find any warnings after configuring clangd with MSVC in linux

### DISCUSSION THREAD

**Discussion by SujjithS** [clangd/clangd#1789] [Q&A]
Below is settings.json I am using inside .vscode  
  "clangd.path": "/usr/bin/clangd",
  "clangd.onConfigChanged": "restart",
  "clangd.arguments": [
      "--compile-commands-dir=${workspaceFolder}/software/business_logic",
      "--clang-tidy",
      "--background-index",
      "--log=info",
      "--pretty",
      "--clang-tidy",
      "--clang-tidy-checks=performance-*,bugprone-*",
      "--completion-style=detailed",
  ],
  "cmake.copyCompileCommands": "${workspaceFolder}/software/business_logic/compile_commands.json",
  "C_Cpp.intelliSenseEngine": "disabled",
  "c-cpp-flylint.clang.enable": false
  
  
  When I check the help>Toggle developer tools , I could observe the message stating "[Extension Host] Clang Language Server is now active!"  But for some reason I dont see any warning messages which goes against the defined rules in .clangd file.

**Comment by SujjithS** [comment]
![Uploading Screenshot from 2023-10-17 17-17-16.png…]()

**Comment by HighCommander4** [comment]
What's an example warning you're expecting to see? Can you show a code example?

**Reply by SujjithS** [reply]
```c++      
#include "algo.hpp"

#include <cstdint>

std::int32_t algo::add(std::int32_t f_a, std::int32_t f_b)
{
    return f_a + f_b;

    
    f_a = f_b; // MISRA 0-1-1 "unreachable-code" violation is expected here, but it doesnt show.
}
```

**Reply by SujjithS** [reply]
The 'clangd' language server was not found on your PATH. Would you like to download and install clangd 16.0.2?

It keeps on saying the above message everytime even if install it.

**Reply by SujjithS** [reply]
The clangd binary '/usr/bin/clangd' was not found. Would you like to download and install clangd 16.0.2?

**Reply by HighCommander4** [reply]
I'm a bit confused as to what this issue is about, since the original comment suggests clangd is starting successfully, but the most recent one talks about the clangd binary not being found.

If you're getting an error about the clangd binary not being found and vscode's auto-install feature doesn't resolve it, you can try downloading a clangd package manually from https://github.com/clangd/clangd/releases and pointing vscode to the binary using the `"clangd.path"` setting.
